client/src/components/Home/HomeSuggestion.js

```diff
@@ -37,8 +37,9 @@ class HomeSuggestion extends Component {
                 <div className="make-gray left-15">Followed by {suggestion.followers.length} people</div>
               </div>
             </div>
+            {/* Follow component takes suggestion object as props */}
             <Follow
-              userId={suggestion._id}
+              user={suggestion}
```

Comment:
in Follow, it is taking userId from props, not user

Reply:
Yeah for get to change it